client/forms.py

```diff
@@ -20,6 +20,7 @@
 onlyNames = [a["name"] for a in data]
 onlyNames.sort()
 CITIES = [(str(x), str(x)) for x in onlyNames]
+CITIES.append(('', ''))
```

Comment:
shouldn't it be the 1st?
plus, add a comment that explains it.

Reply:
Why should it be first? I don't think it matters

Reply:
Users should only search one way of the list, preferably top-down (more intuitive).